x/dkim/client/cli/tx.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+
+	"github.com/burnt-labs/xion/x/dkim/types"
+)
+
+// !NOTE: Must enable in module.go (disabled in favor of autocli.go)
+
+// NewTxCmd returns a root CLI command handler for certain modules
+// transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      types.ModuleName + " subcommands.",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		MsgUpdateParams(),
```

Comment:
This will need all of the possible Tx types implemented

Reply:
Yes they are